styx-common/src/main/java/com/spotify/styx/storage/BigtableStorage.java

```diff
@@ -177,31 +159,6 @@ private SequenceEvent parseEventResult(Result r) throws IOException {
     return SequenceEvent.parseKey(key, event, timestamp);
   }
 
-  private void storeWithRetries(RunnableWithException<IOException> storingOperation) throws IOException {
-    int storeRetries = 0;
-    boolean succeeded = false;
-
-    while (storeRetries < MAX_BIGTABLE_RETRIES && !succeeded) {
-      try {
-        storingOperation.run();
-        succeeded = true;
-      } catch (ResourceNotFoundException e) {
```

Comment:
This doesn't throw this anymore so callers have to see for themselves if `e.getCause()` is a `ResourceNotFoundException`. Kind of clunky.